app/components/publications/shortlist/publications-table-row.hbs

```diff
@@ -16,16 +23,16 @@
   </AuPill>
 </td>
 <td class="auk-table__col--1">
-  {{#each this.mandateePersonNames as |name|}}
-    {{name}}
+  {{#if this.mandateePersonNames}}
+    {{join ", " this.mandateePersonNames}}
   {{else}}
     -
-  {{/each}}
+  {{/if}}
 </td>
 <td class="auk-table__col--1">
   {{date this.decisionDate}}
 </td>
-<td class="auk-table__col--1">
+<td class="auk-table__col--1 u-hide-on-print">
```

Comment:
column was still showing on print. might need input for @brenner-company for this.
I used a class from `wu-legacy.scss`